conjure-java-core/src/test/java/com/palantir/conjure/java/TestBase.java

```diff
@@ -52,7 +52,7 @@ protected static void validateGeneratorOutput(List<Path> files, Path outputDir)
     protected static void validateGeneratorOutput(List<Path> files, Path outputDir, String suffix) throws IOException {
         for (Path file : files) {
             Path output = outputDir.resolve(file.getFileName() + suffix);
-            if (Boolean.valueOf(System.getProperty("recreate", "false"))) {
+            if (true) {
```

Comment:
Don't think you meant to commit this.

Easiest way to trigger this is using `./gradlew test -Drecreate=true`

Reply:
I like doing this from IntelliJ, and for some reason it wasn't taking the system property...